http-server-jetty/src/test/groovy/io/micronaut/servlet/jetty/JettyJsonBodyBindingSpec.groovy

```diff
@@ -358,7 +358,7 @@ class JettyJsonBodyBindingSpec extends Specification {
         }
 
         @Post("/single")
-        Single<String> single(@Body Single<String> message) {
+        Mono<String> single(@Body Mono<String> message) {
```

Comment:
Instead of `Mono<String> single(@Body Mono<String> message) {`

Use `@SingleResult Publisher<String> single(@Body Publisher<String> message){`

Reply:
This changes the body from `{"message":"foo"}` to `[{"message":"foo"}]`, which breaks the test. I can fix the test, but is this what we want from `@SingleResult`?